Panic when a node is offline

```
I0920 05:38:21.895435       1 instances.go:111] instances.InstanceMetadata() called, node: talos62
E0920 05:38:21.899828       1 runtime.go:79] Observed a panic: &runtime.TypeAssertionError{_interface:(*abi.Type)(0x1da8720), concrete:(*abi.Type)(nil), asserted:(*abi.Type)(0x1ccdf60), missingMethod:""} (interface conversion: interface {} is nil, not float64)
goroutine 295 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic({0x1e27840?, 0xc000a00960})
 /go/pkg/mod/k8s.io/apimachinery@v0.29.4/pkg/util/runtime/runtime.go:75 +0x85
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xc000633ad0?})
 /go/pkg/mod/k8s.io/apimachinery@v0.29.4/pkg/util/runtime/runtime.go:49 +0x6b
panic({0x1e27840?, 0xc000a00960?})
 /usr/local/go/src/runtime/panic.go:914 +0x21f
github.com/sergelogvinov/proxmox-cloud-controller-manager/pkg/proxmox.(*instances).getInstanceType(0xc00077e240?, 0xa?, {0xc00062ac22, 0x4})
 /src/pkg/proxmox/instances.go:209 +0x234
github.com/sergelogvinov/proxmox-cloud-controller-manager/pkg/proxmox.(*instances).InstanceMetadata(0xc00069a408, {0x0?, 0x0?}, 0xc000b0f600)
 /src/pkg/proxmox/instances.go:149 +0x7a7
k8s.io/cloud-provider/controllers/node.(*CloudNodeController).getInstanceNodeAddresses(0xc0009a6ab0, {0x24aca30, 0x36bae80}, 0xc000b0f600)
 /go/pkg/mod/k8s.io/cloud-provider@v0.29.4/controllers/node/node_controller.go:655 +0x5f
k8s.io/cloud-provider/controllers/node.(*CloudNodeController).UpdateNodeStatus.func2(0x8116eab67a69ba8a?)
 /go/pkg/mod/k8s.io/cloud-provider@v0.29.4/controllers/node/node_controller.go:279 +0x289
k8s.io/client-go/util/workqueue.ParallelizeUntil.func1()
 /go/pkg/mod/k8s.io/client-go@v0.29.4/util/workqueue/parallelizer.go:90 +0xfa
created by k8s.io/client-go/util/workqueue.ParallelizeUntil in goroutine 203
 /go/pkg/mod/k8s.io/client-go@v0.29.4/util/workqueue/parallelizer.go:76 +0x1d6
I0920 05:38:21.899870       1 node_controller.go:267] Update 6 nodes status took 25.57241ms.
panic: interface conversion: interface {} is nil, not float64 [recovered]
 panic: interface conversion: interface {} is nil, not float64
goroutine 295 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xc000633ad0?})
 /go/pkg/mod/k8s.io/apimachinery@v0.29.4/pkg/util/runtime/runtime.go:56 +0xcd
panic({0x1e27840?, 0xc000a00960?})
 /usr/local/go/src/runtime/panic.go:914 +0x21f
github.com/sergelogvinov/proxmox-cloud-controller-manager/pkg/proxmox.(*instances).getInstanceType(0xc00077e240?, 0xa?, {0xc00062ac22, 0x4})
 /src/pkg/proxmox/instances.go:209 +0x234
github.com/sergelogvinov/proxmox-cloud-controller-manager/pkg/proxmox.(*instances).InstanceMetadata(0xc00069a408, {0x0?, 0x0?}, 0xc000b0f600)
 /src/pkg/proxmox/instances.go:149 +0x7a7
k8s.io/cloud-provider/controllers/node.(*CloudNodeController).getInstanceNodeAddresses(0xc0009a6ab0, {0x24aca30, 0x36bae80}, 0xc000b0f600)
 /go/pkg/mod/k8s.io/cloud-provider@v0.29.4/controllers/node/node_controller.go:655 +0x5f
k8s.io/cloud-provider/controllers/node.(*CloudNodeController).UpdateNodeStatus.func2(0x8116eab67a69ba8a?)
 /go/pkg/mod/k8s.io/cloud-provider@v0.29.4/controllers/node/node_controller.go:279 +0x289
k8s.io/client-go/util/workqueue.ParallelizeUntil.func1()
 /go/pkg/mod/k8s.io/client-go@v0.29.4/util/workqueue/parallelizer.go:90 +0xfa
created by k8s.io/client-go/util/workqueue.ParallelizeUntil in goroutine 203
 /go/pkg/mod/k8s.io/client-go@v0.29.4/util/workqueue/parallelizer.go:76 +0x1d6

```

talos62 is offline. I believe these are the offending lines:

https://github.com/sergelogvinov/proxmox-cloud-controller-manager/blob/2b6435273f3fec338578c11a53d8bc859b80c02b/pkg/proxmox/instances.go#L238-L241

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,9 +1,9 @@
-# syntax = docker/dockerfile:1.5
+# syntax = docker/dockerfile:1.10
 ########################################
 
 FROM --platform=${BUILDPLATFORM} golang:1.23.1-alpine3.20 AS builder
 RUN apk update && apk add --no-cache make
-ENV GO111MODULE on
+ENV GO111MODULE=on
 WORKDIR /src
 
 COPY go.mod go.sum /src

--- a/Makefile
+++ b/Makefile
@@ -21,7 +21,8 @@ TESTARGS ?= "-v"
 
 BUILD_ARGS := --platform=$(PLATFORM)
 ifeq ($(PUSH),true)
-BUILD_ARGS += --push=$(PUSH) --output type=image,annotation-index.org.opencontainers.image.source="https://github.com/$(USERNAME)/$(PROJECT)"
+BUILD_ARGS += --push=$(PUSH)
+BUILD_ARGS += --output type=image,annotation-index.org.opencontainers.image.source="https://github.com/$(USERNAME)/$(PROJECT)",annotation-index.org.opencontainers.image.description="Proxmox VE CCM for Kubernetes"
 else
 BUILD_ARGS += --output type=docker
 endif

--- a/pkg/proxmox/instances.go
+++ b/pkg/proxmox/instances.go
@@ -235,6 +235,10 @@ func (i *instances) getInstanceType(vmRef *pxapi.VmRef, region string) (string, 
 		return "", err
 	}
 
+	if vmInfo["maxcpu"] == nil || vmInfo["maxmem"] == nil {
+		return "", fmt.Errorf("instances.getInstanceType() failed to get instance type")
+	}
+
 	return fmt.Sprintf("%.0fVCPU-%.0fGB",
 		vmInfo["maxcpu"].(float64),
 		vmInfo["maxmem"].(float64)/1024/1024/1024), nil

